jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/file/S3Uploader.java

```diff
@@ -86,11 +87,17 @@ public void uploadFiles(String s3Bucket,String dataFolder, String s3Region,Strin
 	 * @param to the to
 	 */
 	public void backUpFiles(String s3Bucket,String s3Region,String from,String to){
```

Comment:
same here - instead of returning `void` method could return a status object. and then inside `orchestrate` method all of such objects are consolidated and decision would be made. What do you think about that?

Reply:
here we are incrementing eCount and not setting jobStatus to failed in this step. Only data alert will be created if error occurs but job will not be ended in failed status for this method.